### DISCUSSION TITLE
v0.8 designs

### DISCUSSION THREAD

**Discussion by monicakogler** [sofn-xyz/mailing#140] [Show and tell]
Hey y'all,

We wanted to share a preview of what we're planning to build next for those who might be interested. In the screenshots below, you'll notice:

- Dark mode #82 
- File list and previews in the same view #145 
- Expanded/compact view of file list #147  #146 
- Analytics

Thanks in advance for any thoughts and feedback. ❤️

![expanded – preview@2x](https://user-images.githubusercontent.com/609038/186563600-e146a17c-c66b-4780-8448-59a237250958.jpg)

![compact – analytics@2x](https://user-images.githubusercontent.com/609038/186563595-c7ed95c7-4091-49db-a732-174690e3f3dc.jpg)

**Comment by psugihara** [comment]
Thx @monicakogler! Excited for this much improved UI. I'll edit your post to link to issues as they're created.

My only big addition is that the actual analytics pieces will be a bigger lift and moved to 0.9.0, but a placeholder in the header feels good.

And since we're here talking 0.8.0 roadmap, my non-design wishlist:
- `mailing deploy` to make deployment even easier via a generated vercel.json
- Anonymous telemetry so that we understand what features/commands people are actually using and can prioritize work better #143 
- Revisit default templates (in particular TextEmail is probably too broad, love what you have in the mocks above)

**Comment by mrlubos** [comment]
Love love love this!

**Comment by mrlubos** [comment]
Feel free to turn these into issues, a couple new thoughts.

I think it might be useful to store template descriptions such as "this template gets sent when X happens." This is currently annoying to manage cleanly.

The other minor thing is the ".tsx" showing in email templates. I quite like the approach Storybook has where you can customise how story titles appear.

**Reply by monicakogler** [reply]
Oh yeah, great call re: email trigger. Had that in earlier versions but it fell through the cracks in this latest rev. I'll start thinking about how to incorporate. Ty ty!

Defer to @psugihara / @alexfarrill on ".tsx" ...

**Reply by mrlubos** [reply]
Looking at this further @monicakogler, is there a way to hide the side panel?

**Reply by monicakogler** [reply]
Not yet, could be cool if you could drag and collapse it. We'll see how it feels...

**Reply by mrlubos** [reply]
@monicakogler personally, not a fan of dragging in these interfaces. The use case I was thinking of was a full screen view to show off my templates. Or eventually, mailing could have a dev server you could connect to from your phone to test the template on a real mobile device, and you wouldn't want the side panel to take up space in that layout

**Reply by monicakogler** [reply]
Got it, thanks. That makes sense. You thinking a button to hide the panel would be nicer? And/or maybe even a way to hide all the UI a la Figma? In any case, we're planning to hide the panel behind a hamburger for mobile. 

<img width="1254" alt="Screen Shot 2022-08-27 at 8 07 28 AM" src="https://user-images.githubusercontent.com/609038/187036039-bbec3fda-6848-4665-bb99-c6f24f92dd5e.png">

**Reply by mrlubos** [reply]
No preference. This mobile UI looks good!

**Reply by monicakogler** [reply]
fyi, this feature is live in the latest version with cmd + .

let us know if you have any feedback :)
